### ISSUE TITLE
Robots.txt do not refresh, keeps showing local

### ISSUE THREAD

**Issue by arentsen** [nystudio107/craft-seomatic#850]
### Questions

1. My robots.txt does not refresh (nor does human.txt), regardless of what I enter. It keeps showing local, even in the production environment. I changed 'local' to dev and 'live' to production, but I assume that should not be an issue?
2. The robots.txt gives the naked domain even though the siteUrl is de.karstenwitt.com and en.karstenwitt.com. Also both subdomains show the same robots.txt (see below)
3. The main sitemap does not load in the [German](https://de.karstenwitt.com/sitemap.xml) website, but does in the [English](https://en.karstenwitt.com/sitemap.xml) one 

### Additional context
https://de.karstenwitt.com/robots.txt or https://en.karstenwitt.com/robots.txt

```
# robots.txt for https://karstenwitt.com/
Sitemap: https://karstenwitt.com/sitemaps-1-sitemap.xml
# local - disallow all
User-agent: *
Disallow: /
```

**Comment by khalwat** [comment]
If it's showing `local` probably the `ENVIRONMENT` variable in your `.env` is set to something like `dev` ?

When you say "I changed 'local' to dev and 'live' to production" can you explain what you mean here?

**Comment by arentsen** [comment]
My `.env` file says `ENVIRONMENT=production`. However, the robots.txt does not even update when I add parts to it or remove everything but `default`. It simply does not change. UPDATE: I found out there was a robot.txt in the web directory, which I did not remember putting there. It is updating now.

I used "dev" in my .env file instead of "local", and "production" instead of "live". But I experimented a bit now and I see that SEOmatic automatically recognizes `production` and outputs the "Live" robots.txt. Great plugin, thanks so much for writing it. A great feature is `???`!
